database/index.js

```diff
@@ -51,16 +51,28 @@ const insertAvailability = (availability, count, callback) => {
 
 const updateAvailability = (availability, count, callback) => {
   console.log(availability);
-  connection.query('INSERT INTO availability (availability_date, price, is_available, listings_id) VALUES (?, ?, ?, ?)', [availability.date, availability.price, availability.is_available, availability.listings_id], function (err, results, fields) {
+  connection.query(`SELECT * from availability INNER JOIN listings ON listings.id = ${availability.listings_id} where availability.availability_date = '${availability.date}' ORDER BY availability.id DESC LIMIT 1`, (err, listingsResults, fields) => {
```

Comment:
Is the LIMIT 1 necessary if each record is unique for a listings_id+availability_date combination? (disregard if my assumption is incorrect)

Reply:
@christine-w the LIMIT 1 is necessary because I'm keeping a history of availability for listings_id+availability_date combinations and want to query the most recent update